specification/appcomplianceautomation/resource-manager/Microsoft.AppComplianceAutomation/preview/2022-11-16-preview/appcomplianceautomation.json

```diff
@@ -0,0 +1,1374 @@
+        "operationId": "Report_CreateOrUpdate",
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "Parameters for the create or update operation",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ReportResource"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ReportNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The response describes a AppComplianceAutomation report.",
+            "schema": {
+              "$ref": "#/definitions/ReportResource"
+            }
+          },
+          "201": {
+            "description": "Created. The response describes the new service and contains a Location header to query the operation result.",
+            "schema": {
+              "$ref": "#/definitions/ReportResource"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "Report_CreateOrUpdate": {
+            "$ref": "./examples/Report_CreateOrUpdate.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "AppComplianceAutomation"
+        ],
+        "description": "Update an exiting AppComplianceAutomation report.",
+        "operationId": "Report_Update",
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "Parameters for the create or update operation",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ReportResourcePatch"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ReportNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The response describes a AppComplianceAutomation report.",
+            "schema": {
+              "$ref": "#/definitions/ReportResource"
+            }
+          },
+          "201": {
+            "description": "Long running operation support.",
+            "schema": {
+              "$ref": "#/definitions/ReportResource"
+            }
```

Comment:
Long running PATCH should return 202 : 

[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#updating-using-patch](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#updating-using-patch)

Reply:
@raosuhas Yes, we implemented the code which will return 202. We can fix it in the API specs.

Reply:
Hi @raosuhas ~
The statusCode has been changed from 201 to 202. See [here](https://github.com/Azure/azure-rest-api-specs/pull/21152/files#diff-16cc0066db31d47ee8b4f0e9385ece034881677d74a14b4d48790444b24bda4dR245).

Reply:
After testing we found that RPSaas would change our 202 statuscode to 201. See [here.](https://stackoverflow.microsoft.com/questions/311423)
So we keep both 202 and 201 in the spec.